### PR TITLE
CHE-55: Adding kubernetes-client / kubernetes-model dependencies. Sorting pom.xml

### PR DESCRIPTION
**Pull request by ibuziuk**

### CODE REVIEW

**Comment by ibuziuk** [comment]
@l0rd could you please apply ?

File: plugins/plugin-docker/che-plugin-docker-client/pom.xml [hunk 1]
```diff
@@ -52,6 +52,24 @@
             <artifactId>guice</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-client</artifactId>
+            <version>1.4.26</version>
```
Comment (skabashnyuk) [inline plugins/plugin-docker/che-plugin-docker-client/pom.xml:61]: move this variable to https://github.com/eclipse/che-dependencies/blob/master/pom.xml

File: plugins/plugin-docker/che-plugin-docker-client/pom.xml [hunk 2]
```diff
@@ -52,6 +52,24 @@
...
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-model</artifactId>
```
Comment (skabashnyuk) [inline plugins/plugin-docker/che-plugin-docker-client/pom.xml:65]: https://github.com/eclipse/che-dependencies/blob/master/pom.xml

**Review by skabashnyuk — CHANGES_REQUESTED** [review]
Need crosslink to https://github.com/eclipse/che-dependencies/blob/master/pom.xml and CQs

**Comment by ibuziuk** [comment]
@skabashnyuk ok, lets probably wait for https://github.com/eclipse/che-dependencies/pull/23 been merged

**Comment by ibuziuk** [comment]
not relevant any more. Closing

**Pull request closed without merging**